src/core/gps/qgsnmeaconnection.cpp

```diff
@@ -115,7 +115,7 @@ void QgsNmeaConnection::processStringBuffer()
           mStatus = GPSDataReceived;
           QgsDebugMsgLevel( QStringLiteral( "*******************GPS data received****************" ), 2 );
         }
-        else if ( substring.startsWith( QLatin1String( "$GPVTG" ) ) )
+        else if ( substring.startsWith( QLatin1String( "$GPVTG" ) ) || substring.startsWith( QLatin1String( "$GNVTG" ) ) )
```

Comment:
This won't have any impact unless you also modify `nmea_parse_GPVTG`. See e.g. 299c61292b58602c4d6f59628b3bafd6125abd19 for an example on what you'd need to change in parse.c here!

Reply:
Thanks @nyalldawson 
I have only read the G?GGA parser and extrapolated that the others would have been patched also.
As noted in earlier comments, not completely convinced for failing on `type not in 'PN'` but kept the style for consistency.